### PR TITLE
Allow more time for members to comment on PRs

### PR DESCRIPTION
**Pull request by derrell**

### CODE REVIEW

**Review by cboulanger — APPROVED** [review]

File: merge-process.md [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
 Acceptance of a PR is accomplished as follows:
 
-- When the stream of comments has died down -- in particular, dissenting
-  comments -- a member of the commit team will request reviews from the other
-  team members.
 - If the PR proposer is a member of the commit team, he/she does not have voting
   rights on this PR.
-- A PR will be accepted when one of the following conditions has been met:
+- A PR will be accepted when at least three business days have passed since the call 
+  for votes, and one of the following conditions has been met:
```
Comment (level420) [inline merge-process.md]: I think we should use "since review started" instead of "since the call for votes"

File: merge-process.md [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
   - PR contains only documentation changes
     - two assenting votes have been cast with no dissenting votes.
   - PR contains any code changes
-    - five assenting votes have been cast with no dissenting votes
-    - at least two business days have passed since the call for votes, and two
-      assenting votes have been cast with no dissenting votes
-- If there are dissenting votes during the voting process, discussion will ensue
-  and a new vote will be called when the points of conflict have been resolved.
+    - five assenting votes have been cast with no dissenting votes.
```
Comment (level420) [inline merge-process.md]: We should use the github review wording here "approval" and "request for changes".
And we could explain that adding a change request blocks merging.

**Review by level420 — CHANGES_REQUESTED** [review]

**Review by level420 — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by level420 — CHANGES_REQUESTED** [review]
I noticed just now that the passage `at least two business days have passed since the call for votes, and two assenting votes have been cast with no dissenting votes` has been removed completely as a positive requirement "signal" for merging.

From my point of view we still need something similar, otherwise we will have PRs with 4 approvals which never get merged because the core team members are busy or simply forgot about the review.

What about: 
`at least 5 business days have passed since review request, and 3 assenting votes (approvals) have been cast with no dissenting votes (request for changes)`

**Comment by derrell** [comment]
If we can't get 5 approvals, I'm not sure a PR should be merged. If we happen to hit a time when many core members are away, the PR may just have to wait a week or more before merging, but I still think it's worthwhile allowing people to comment even if on holiday for a while. It's not like (in the vast majority of cases), people can't use the changes from the PR in their own development during that period...

**Comment by level420** [comment]
@derrell OK! convinced.

**Review by level420 — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by cajus — APPROVED** [review]

**Pull request merged**